modules/EnsEMBL/Web/Component/Shared.pm

```diff
@@ -25,712 +25,12 @@ package EnsEMBL::Web::Component::Shared;
 use strict;
 
 use HTML::Entities  qw(encode_entities);
-use Text::Wrap      qw(wrap);
```

Comment:
I don't think your branch is up-to-date - there are a load of changes here that are from my code refactoring, and are nothing to do with the mobile site.

Reply:
Yes, I have forgotten to update the base branch before branching. I'll recreate the branch and the PR.